.circleci/workflows.yml

```diff
@@ -603,27 +577,46 @@ jobs:
 
   # Test VMs
 
-  aarch64-linux-generic-test-vm:
+  ferrocenecoretest-generic-test:
     executor: linux-vm
     parameters:
+      qemu-arch:
+        type: string
+      qemu-cpu:
+        type: string
+      target:
+        type: string
       job:
         type: string
       resource-class:
         type: string
     resource_class: << parameters.resource-class >>
     environment:
-      FERROCENE_HOST: ""
-      FERROCENE_TARGETS: aarch64-unknown-ferrocenecoretest
-      # See ferrocene/ci/split-tasks.py for a list of tasks executed by this.
+      FERROCENE_HOST: x86_64-unknown-linux-gnu
+      FERROCENE_TARGETS: << parameters.target >>
       SCRIPT: |
-        TEST_DEVICE_ADDR=127.0.0.1:12345 ./x.py --stage 2 test $(ferrocene/ci/split-tasks.py << parameters.job >>) --ferrocene-test-one-crate-per-cargo-call
-
+        sudo update-binfmts --import && sudo update-binfmts --enable << parameters.qemu-arch >> && QEMU_CPU=<< parameters.qemu-cpu >> ./x.py --stage 2 test $(ferrocene/ci/split-tasks.py << parameters.job >>) --ferrocene-test-one-crate-per-cargo-call
```

Comment:
I think you can split this into multiple lines and it'll bne ok. 🤔

Reply:
I feel like I have tried that before (on a different PR) and it failed so I had to revert it. now I'm scared to try :sweat_smile:  . maybe when the "ferrocene-ci.wrapper" is used you can't use newlines?

Reply:
Ok. :)